test/fixtures/books.yml

```diff
@@ -6,6 +6,7 @@ becoming:
   release_date: 2018-02-01
   status: 'available'
   user: schmidt
+  borrower_id: nil
```

Comment:
You could have a default value for this attribute, defined at the top of the fixture file.

Reply:
done ✅